### DISCUSSION TITLE
Custom data in multiple columns

### DISCUSSION THREAD

**Discussion by hana-krizova** [kagkarlsson/db-scheduler#415] [Q&A] [answered]
Hi Kagkarlsson and community :-),

is any nice possibility of storing custom task data in multiple columns?

Rationale: Custom task data can contain multiple things - for example if user want to suspend execution, last error, progress in percentages. If all these things are stored in one column (task_data), then the database does not satisfy the first normal form (1NF).

Thank you very much for db-scheduler!
Hana Křížová

**Comment by kagkarlsson** [comment, marked as answer]
Hi! That is not possible no. Data goes into a single column
